Balancer-Bots/Pool-Registered/src/agent.spec.ts

```diff
@@ -0,0 +1,132 @@
+import { ethers, Finding, FindingSeverity, FindingType, HandleBlock } from "forta-agent";
+import { createAddress, MockEthersProvider, TestBlockEvent } from "forta-agent-tools/lib/tests";
+import { provideHandleBlock } from "./agent";
+import { POOL_REGISTERED_ABI } from "./constants";
+import { AgentConfig, PoolSpecialization } from "./utils";
+
+const createChecksumAddress = (address: string): string => ethers.utils.getAddress(createAddress(address.toString()));
+const createBytes32 = (value: string): string => ethers.utils.hexZeroPad(value, 32);
```

Comment:
there is `ethers.utils.formatBytes32String` function that returns a bytes32 string representation of text. it can be useful here

Reply:
Yes, thanks, I used it in another bot. It's a bit different, because `"1"` in `ethers.utils.formatBytes32String()` would get the ASCII value of `'1'` instead of the actual number, but in this case both would work, so I think keeping this like this is fine.